scripts/exregional_make_lbcs.sh

```diff
@@ -525,7 +526,9 @@ located in the following directory:
 # the forecast hour of the FV3-LAM (which is not necessarily the same as
 # that of the external model since their start times may be offset).
 #
-  fcst_hhh_FV3LAM=$( printf "%03d" "${LBC_SPEC_FCST_HRS[$i]}" )
+  lbc_spec_fhrs=( "${EXTRN_MDL_LBC_SPEC_FHRS[$i]}" ) 
+  fcst_hhh=$(( ${lbc_spec_fhrs} - time_offset_hrs ))
```

Comment:
As I mentioned in the JOB above, please use the global variable directly here.

Reply:
OK